src/Components/GrowthChart/GrowthChartOptions/GrowthChartAnnotations.tsx

```diff
@@ -0,0 +1,47 @@
+import i18n from '@dhis2/d2-i18n';
+
+export interface AnnotationLabel {
+    display: boolean;
+    content?: () => string;
+    position?: 'top' | 'bottom' | 'center' | 'start' | 'end';
+    yAdjust?: number;
+}
+
+interface Annotation {
+    display: boolean;
+    type: 'line';
+    scaleID?: string;
+    value?: number;
+    borderWidth?: number;
+    label?: AnnotationLabel;
+}
+
+export const GrowthChartAnnotations = (xAxisValues: number[]) => {
+    const annotations: Annotation[] = [];
+
+    xAxisValues.forEach(
+        (label, index) => {
+            if (label % 12 === 0 && label !== 0) {
+                annotations.push({
```

Comment:
The map function automatically returns a new array, so there is no need to declare an empty array first and then use push to fill the array. Therefore, I have a suggestion:
```suggestion
    .filter((label) => label % 12 === 0 && label !== 0)
        .map((label) => ({
```